iOS/TvShows/TvShows/Common/Services/UserService.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import RxSwift
+
+class UserService {
+
+    static let shared = UserService()
+
+    private let apiClient: ApiClientProtocol
+    private let userDefaults: UserDefaults
```

Comment:
Service around user defaults would also be nice but rather focus on Rx.

Reply:
Deleted it, I will add it once I need it.